.github/workflows/publish_release.yml

```diff
@@ -0,0 +1,146 @@
+name: Publish NovelRT
+
+on:
+  workflow_dispatch:
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+
+    outputs:
+      tag: ${{ steps.tag_version.outputs.new_tag }}
+      id: ${{ steps.create_release.outputs.id }}
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - uses: mathieudutour/github-tag-action@v6.0
+      id: tag_version
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - uses: ncipollo/release-action@v1
+      id: create_release
+      with:
+        tag: ${{ steps.tag_version.outputs.new_tag }}
+        name: NovelRT ${{ steps.tag_version.outputs.new_tag }}
+        body: ${{ steps.tag_version.outputs.changelog }}
+        prerelease: true
+        draft: true
+
+  build-windows:
```

Comment:
You can merge this with build-ubuntu using conditions: <https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution>

I'd do something like `run: cibuild.cmd if: ${{ startsWith(matrix.os, 'windows') }}`

Reply:
The only issue I ran into with this was that I could not get the container to comply with the `if` statement to only be used with ubuntu (GH was stating I had a syntax error?)

Reply:
Interesting. The syntax I suggested *should* work.

Reply:
Also, if I'm being honest, since we're going through all of this effort to simplify our CI pipeline, we should probably just remove the build scripts entirely. I seriously doubt that we're going to migrate away from GitHub Actions in the near future, and reproducing a CI environment is already possible using something like [act](https://github.com/nektos/act).